sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -667,4 +672,19 @@ export class StudentService extends RecordDataModelService<Student> {
       entityManager,
     );
   }
+
+  /**
+   * Get student PD status.
+   * @param pdVerified sfas pdVerified flag.
+   * @returns PD status.
```

Comment:
Instead of `PD status.`, maybe we can give something as `disability status`, same for line 677, to keep everything generic